src/components/NewTodoField/NewTodoField.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useRef } from 'react';
+
+type Props = {
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+  title: string;
+  onTitleChange: (title: string) => void;
```

Comment:
it looks horibble. ) I tried to start React again and debug my HT and found that just every movement can cause rerendering. i want you to know how looks yours when you just push the keys in input. 
![image](https://user-images.githubusercontent.com/31005050/207982282-9799ed5b-0f82-4b6d-8be8-c5e39704a59b.png)